examples/cluster/plot_kmeans_assumptions.py

```diff
@@ -46,15 +48,19 @@
 X_varied, y_varied = make_blobs(
     n_samples=n_samples, cluster_std=[1.0, 2.5, 0.5], random_state=random_state
 )
-y_pred = KMeans(n_clusters=3, random_state=random_state).fit_predict(X_varied)
+y_pred = KMeans(n_clusters=3, n_init="auto", random_state=random_state).fit_predict(
+    X_varied
+)
 
 plt.subplot(223)
 plt.scatter(X_varied[:, 0], X_varied[:, 1], c=y_pred)
 plt.title("Unequal Variance")
 
 # Unevenly sized blobs
 X_filtered = np.vstack((X[y == 0][:500], X[y == 1][:100], X[y == 2][:10]))
-y_pred = KMeans(n_clusters=3, random_state=random_state).fit_predict(X_filtered)
+y_pred = KMeans(n_clusters=3, n_init="auto", random_state=random_state).fit_predict(
+    X_filtered
+)
```

Comment:
```suggestion
y_pred = KMeans(n_clusters=3, n_init=5, random_state=random_state).fit_predict(
    X_filtered
)
```

Reply:
Looking at the results, we were not lucky with the initialization and we obtain a result that is not in line with the comment.

I propose to make 3 individual init and check that the result does not change compared to the version in `main`.

Reply:
I set ` n_init=10` for the two cases where the plot was different to the one in main. I left the other two with `n_init="auto"`. Is that fine, or should I also set them to a specific value?